packages/bruno-app/src/components/RequestPane/QueryEditor/index.js

```diff
@@ -45,7 +45,6 @@ export default class QueryEditor extends React.Component {
       brunoVarInfo: {
         variables: getAllVariables(this.props.collection)
       },
-      theme: this.props.editorTheme || 'graphiql',
```

Comment:
There is no `editorTheme` props being passed to this component on any of its occurrences